modules/apigee_edge_teams/src/Controller/TeamMembersList.php

```diff
@@ -51,21 +52,37 @@ class TeamMembersList extends ControllerBase {
    */
   protected $defaultRoles = [];
 
+  /**
+   * The module handler.
+   *
+   * @var \Drupal\Core\Extension\ModuleHandlerInterface|null
+   */
+  protected $moduleHandler;
+
   /**
    * TeamMembersList constructor.
    *
    * @param \Drupal\apigee_edge_teams\TeamMembershipManagerInterface $team_membership_manager
    *   The team membership manager service.
    * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
    *   The entity type manager.
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface|null $module_handler
+   *   The module handler.
    */
-  public function __construct(TeamMembershipManagerInterface $team_membership_manager, EntityTypeManagerInterface $entity_type_manager) {
+  public function __construct(TeamMembershipManagerInterface $team_membership_manager, EntityTypeManagerInterface $entity_type_manager, ModuleHandlerInterface $module_handler = NULL) {
+    if (!$module_handler) {
+      @trigger_error('Calling ' . __METHOD__ . ' without the $module_handler is deprecated in apigee_edge:8-x-1.19 and is required before apigee_edge:8.x-2.0. See https://github.com/apigee/apigee-edge-drupal/pull/518.', E_USER_DEPRECATED);
+      $module_handler = \Drupal::moduleHandler();
+    }
+
```

Comment:
Let's add the new `$module_handler` parameter to the `create()` function too.

Reply:
@arshad Talked this over with the Apigee team and it looks good. Lets just get that `$module_handler` param in the `create()` function and mark it as ready for review.

Reply:
Done. How did I miss this! 🤪